source/scripts/components/Snackbar/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+import styles from './snackbar.css';
+
+// TODO: add actionText and onActionClick handler (yes/no, undo, etc..)
+class Snackbar extends React.Component {
+	static propTypes = {
+		text: PropTypes.string.isRequired,
+		onTimeout: PropTypes.func, // for closing portals, should be isRequired
+		className: PropTypes.string,
+		showTime: PropTypes.number,
+		actionText: PropTypes.string,
+		onActionClick: PropTypes.func,
+	};
+
+	static defaultProps = {
+		showTime: 2000,
+	}
+
+	state = {
+		hidden: false,
+	}
+
+	componentDidMount() {
+		setTimeout(() => {
+			this.setState({
+				hidden: true,
+			});
+
+			// this.props.onTimeout(); 	// TODO: uncomment this when portals are ready. this is a callback for parent
+		}, this.props.showTime);
+	}
+
+	render() {
+		const classes = classnames(styles.snackbar, {
+			[styles.hidden]: this.state.hidden,
+		});
+
+		return <div className={classes}>{this.props.text}</div>;
```

Comment:
Snackbar is a notification. Please consider [recommendations for building accessible notifications and alerts](https://github.com/andrew--r/ui-developer-tips/tree/master/tips/008-alerts-accessibility).